src/math/Legendre.h

```diff
@@ -28,6 +29,16 @@ std::vector<double> LegendreRoots(int n);
 //! @return vector of the n-2 roots
 std::vector<double> LegendreDerivRoots(int n);
 
+//! @brief computes points and weights for Lobatto quadrature in 1D
+//! @param numIPs number of integration points
+//! @return pair of quadrature weights and points range [-1,1] including boundary points
+std::pair<std::vector<double>, std::vector<double>> ComputeWeightsAndPoints1DLobatto(int nIps);
```

Comment:
Can we move these to a different header (or rename this one)? I would not have guessed that if I want to call `NuTo::Math::Polynomial::ComputeWeightsAndPoints1DGauss(nIps);`, I'll need to include `Legendre.h`.

Reply:
Do you have an idea? Since gaussian type quadrature parameters are related to orthogonal polynomials I thought this is where one should look.

Reply:
Well the first three functions in this file were called `Legendre`, `LegendreRoots` and `LegendreDerivRoots`, all things that could be useful outside of quadrature and clearly related to `Legendre.h`.  So maybe the new functions could live in a `math/Quadrature.h` or `math/GaussQuadrature.h`? :)